definitions/ext-shelled_spa/definition.yml

```diff
@@ -0,0 +1,39 @@
+domain: EXT
+type: SHELLED_SPA
+synthesis:
+  rules:
+
+    - identifier: shellAppName
+      name: shellAppName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: ShelledBrowserInteraction
+
+    - identifier: shellAppName
+      name: shellAppName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: ShelledPageViewTiming
+
+    - identifier: shellAppName
+      name: shellAppName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: ShelledJavaScriptError
+
+  tags:
+    category:
+      multiValue: false
```

Comment:
I don't believe this is correct.

I'm seeing events in `ShelledBrowserInteraction` with the same `shellAppName` and different categories.
This will cause a loop of updates so please convert this tag into `multiValue: true`